recipe_scrapers/_utils.py

```diff
@@ -311,3 +312,20 @@ def change_keys(obj, convert):
 def get_url_slug(url):
     path = url_path_to_dict(url).get("path")
     return path.split("/")[-1]
+
+
+def get_abstract_methods():
+    from ._abstract import AbstractScraper
+
+    methods = []
+    source_lines = inspect.getsourcelines(AbstractScraper)[0]
+    for line in source_lines:
+        if line.strip().startswith("def "):
+            method_name = line.split("def ")[1].split("(")[0].strip()
+            if not method_name.startswith("_") and method_name not in [
+                "links",
+                "to_json",
+            ]:
+                methods.append(method_name)
+
+    return methods
```

Comment:
+1 for using the `inspect` module, but it would be nice to avoid having to read/compare the source code lines.

I'd suggest:

```suggestion
    return [
        name for name, value in AbstractScraper.__dict__.items()  # Attributes of the abstract scraper class..
        if not name.startswith("_")  # ... that are not private ...
        and inspect.isfunction(value) # ... and are functions ...
        and name not in {"links", "to_json"}  # ... and not excluded as special-cases.
   ]
```

Reply:
I've implemented this change but added 
```python
        or name == "host" # ... explicitly include the `host` method.
```
as the current implementation skipped the classmethod object

Reply:
Ok, yep - makes sense, thank you.